packages/wonder-blocks-layout/src/components/media-layout.tsx

```diff
@@ -138,15 +138,10 @@ class MediaLayoutInternal extends React.Component<CombinedProps, State> {
         return DEFAULT_SIZE;
     }
 
-    // We assume that we're running on the server (or, at least, an unsupported
-    // environment) if there is no window object or matchMedia function
-    // available.
-    isServerSide() {
-        return (
-            Server.isServerSide() ||
-            typeof window === "undefined" ||
-            !window.matchMedia
-        );
+    // We assume that we're running on an unsupported environment) if there is
+    // no window object or matchMedia function available.
+    isUnsupportedEnvironment() {
```

Comment:
**note:** Renamed to reflect the purpose. I'm not certain we need this anymore, but wanted to keep the changes minimal.